specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -1330,22 +1333,22 @@ union AcsMessageChannelKind {
 /** Message delivery status */
 union AcsMessageDeliveryStatus {
   /** Read */
-  "read",
+  Read: "read",
 
   /** Delivered */
-  "delivered",
+  Delivered: "delivered",
```

Comment:
Why did we add the keys here but remove in the other model?

Reply:
Added the keys here b/c in this union I got twisted around on the swagger def here: https://github.com/Azure/azure-rest-api-specs/blob/ad4acf68519ce36b865e72098bbe59112d6f4a8e/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L1629. The on the wire enum has uppercase, but values in x-ms-enum has lowercase

I will add keys to each union to keep it consistent